Modules/_hashopenssl.c

```diff
@@ -455,6 +456,13 @@ EVPnew(PyObject *name_obj,
         return NULL;
     }
 
+    /*
+     * Load config file, and other important initialisation in order to detect
+     * architecture optimizations correctly. E.g: POWER8 optimization
+     *
+     * Source: https://wiki.openssl.org/index.php/Libcrypto_API
+     */
+    OPENSSL_config(NULL);
```

Comment:
Sure! I tested and I still have the results I expected.
However `OPENSSL_config(NULL);` is deprecated so I'll send this change as soon as I figure it out which interface should I use instead.